logstash.go

```diff
@@ -73,8 +74,8 @@ func (a *LogstashAdapter) Stream(logstream chan *router.Message) {
 		}
 		_, err = a.conn.Write(js)
 		if err != nil {
-			log.Println("logstash:", err)
-			continue
+			log.Println("fatal logstash:", err)
+			os.Exit(3)
```

Comment:
Hi, could you apply the changes from my comment? Would like to merge this or #15, who ever fixes their PR first.